docs/troubleshooting.md

```diff
@@ -8,4 +8,4 @@ Infrastructure logs are visible from the `Global` tenant and require `administra
 `kube:admin` by default does not have the correct permissions to be given the admin role.   See the [access control](access-control.md) documentation for additional informations.  You may grant the permissions by:
 ```
 oc adm policy add-cluster-role-to-user cluster-admin kube:admin
-```
\ No newline at end of file
+```
```

Comment:
not sure what changed here... can we do a `git checkout upstream/master -- docs/troubleshooting.md` just to clear this file from the list of changed ones?

Reply:
It is cleared. Thanks for your instruction.